### PR TITLE
Adds logging when gas-exuding seeds are planted

### PR DESCRIPTION
**Pull request by Kammerjunk**
Instead of having to check all strange and modified plants for fingerprints and their seeds for exude_gasses, this'll add a log whenever a seed which exudes gasses is planted.
Should it add an investigation log to the person planting it? I'm not sure. I can do that if it should be added, but I'd have to find out how exactly first. Should be easy, I guess.

### CODE REVIEW

File: code/modules/hydroponics/hydro_tray.dm [hunk 1]
```diff
@@ -206,6 +206,9 @@
 				if(2)
 					var/turf/T = get_turf(src)
 					msg_admin_attack("[key_name(user)] has planted a spreading vine packet. <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[T.x];Y=[T.y];Z=[T.z]'>(JMP)</a>")
+			if(S.seed.exude_gasses)
+				var/turf/T = get_turf(src)
+				msg_admin_attack("[key_name(user)] has planted a packet exuding [english_list(S.seed.exude_gasses)]. <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[T.x];Y=[T.y];Z=[T.z]'>(JMP)</a>")
```
Comment (PJB3005) [inline code/modules/hydroponics/hydro_tray.dm]: Isn't there a standardized proc to do jump to hrefs for admins?

Comment (Kammerjunk) [reply code/modules/hydroponics/hydro_tray.dm]: I dunno I actually just copied the adminjump text for the logs right above (for planting seeds that spread).
I found `[formatJumpTo(T)]` in some logging in Chemistry-Recipes.dm. Would that work?

Comment (PJB3005) [reply code/modules/hydroponics/hydro_tray.dm]: That's it, probably.

**Comment by Kammerjunk** [comment]
new message
Works much better, yeah.

**Review by PJB3005 — APPROVED** [review]

**Comment by BarneyGumball** [comment]
You should use `add_gamelogs(user, "planted a packet exuding [english_list(S.seed.exude_gasses)].", tp_link = TRUE)`.

It handles the jump link, adding it to the logs and messaging admins.

**Comment by Kammerjunk** [comment]
newer message
Sure does. It even adds its own full stop. I dunno if there are any differences other than colour and looking cleaner so I'll just use that one (I've removed the full stop to prevent passive aggressive logging)

**Pull request merged**